### PR TITLE
cpp-lazy: add recipe

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **cpp-lazy/6.0.0**

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/README.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by uilianries — DISMISSED** [review]
LGTM

File: recipes/cpp-lazy/all/conandata.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+sources:
+  "6.0.0":
+    url: "https://github.com/MarcDirven/cpp-lazy/releases/download/v6.0.0/cpp-lazy-src.zip"
+    sha256: "424973eabaab8c756dae5e1795febeface65bb1dc42f474e5e2e1e4ab3effc5e"
+patches:
+  "6.0.0":
+    - patch_file: "patches/0001-support-gcc5.patch"
+      patch_description: "fix compilation errors on gcc5"
+      patch_type: "backport"
```
Comment (prince-chrismc) [inline recipes/cpp-lazy/all/conandata.yml:9]: #14370 Need to fix the linter

```suggestion
      patch_type: "portability"
```

**Review by prince-chrismc — COMMENTED** [review]
Just the patch type :)

**Review by uilianries — APPROVED** [review]
LGTM

**Review by prince-chrismc — APPROVED** [review]

**Pull request merged**